core/src/main/java/com/chopshop166/chopshoplib/sensors/gyro/PigeonGyro2.java

```diff
@@ -24,9 +25,7 @@ public PigeonGyro2(final Pigeon2 gyro) {
         this.gyro = gyro;
 
         // Automatically invert if the gyro is upside-down
-        final short accelerometerXYZ[] = new short[3];
-        this.gyro.getBiasedAccelerometer(accelerometerXYZ);
-        if (accelerometerXYZ[2] < 0) {
+        if (this.gyro.getRoll().getValueAsDouble() < 0) {
```

Comment:
I think we can remove this and just use the on-device calibration to handle this.

Reply:
Does this require a software change, or is it something we do in firmware?

Reply:
The calibration is on the device, no software changes necessary. (Just remove all of this invert code)